requirements-test.txt

```diff
@@ -0,0 +1,2 @@
+pre-commit==0.10.1
+setuptools==34.0.2
```

Comment:
Normal python installation usually contains setuptools, so this is only needed for vagga env. Please remove it from this file and add to vagga config